### ISSUE TITLE
Parameters Object does not list `examples` field

### ISSUE THREAD

**Issue by DavidBiesack** [OAI/OpenAPI-Specification#766]
I looked at #636 again after @darrelmiller posted "Extending representations to parameters and headers" #762 . I think the **[Parameter Object](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#parameterObject)** section should include the (singular) `example` field under **[Fixed Fields](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#fixed-fields-9)**.  `examples` are used in parameters in the samples under **[Examples Object](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#examplesObject)** :

```
# in a parameter

  parameters:
    - name: 'zip'
      in: 'query'
      type: 'string'
      format: 'zip'
      example: 
        $ref: 'http://foo.bar#/examples/zip-example'
```

Also, given the ongoing discussions about multiple representations/content in #762 the `example` or `examples` field should be added to  `content` (was `representations`) content object, as examples and `content` objects should apply to parameters as well.

**Comment by darrelmiller** [comment]
@DavidBiesack It's there https://github.com/OAI/OpenAPI-Specification/pull/761/files#diff-4a97aef5bba9ddb7fd8609f48f4bdc30R906

**Comment by DavidBiesack** [comment]
Thanks @darrelmiller -- is it worth splitting that off into a separate PR, since the `examples` #636 is already merged but the representations/content PR #761 is still a proposal?

**Comment by fehguy** [comment]
Both merged now
